sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/aws.yaml

```diff
@@ -2,46 +2,42 @@ provider:
   metadata:
     name: AWS
     category: Cloud provider
-    references:
-      - description: Best practices for managing AWS access keys
-        link: https://docs.aws.amazon.com/accounts/latest/reference/credentials-access-keys-best-practices.html
-        type: Documentation
-      - description: Managing access keys for IAM users
-        link: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html
-        type: Documentation
-      - description: OWASP Top 10 2021 Category A7 - Identification and Authentication Failures
-        link: https://owasp.org/Top10/A07_2021-Identification_and_Authentication_Failures/
-        type: Standards
-      - description: OWASP Top 10 2017 Category A3 - Sensitive Data Exposure
-        link: https://www.owasp.org/www-project-top-ten/2017/A3_2017-Sensitive_Data_Exposure
-        type: Standards
-      - description: MITRE, CWE-798 - Use of Hard-coded Credentials
-        link: https://cwe.mitre.org/data/definitions/798
-        type: Standards
-      - description: MITRE, CWE-259 - Use of Hard-coded Password
-        link: https://cwe.mitre.org/data/definitions/259
-        type: Standards
-      - description: SANS Top 25 - Porous Defenses
-        link: https://www.sans.org/top25-software-errors/#cat3
-        type: Standards
-
   detection:
     pre:
       include:
         content:
           - aws
           - amazon
-    post:
-      patternNot:
-        - "EXAMPLE"
 
   rules:
     - rspecKey: S6290
       id: aws-secret-access-keys-1
       metadata:
         name: AWS Secret Access Keys
         message: Make sure this AWS Secret Access Key gets revoked, changed, and removed from the code.
+      detection:
+        matching:
+          pattern: "(?is)aws.{0,30}secret.{0,30}\\b([0-9a-z/+]{40})\\b"
+        post:
+          statisticalFilter: 
+            threshold: 4.2
+          patternNot:
+            # Character repeated 5 times
+            - "([\\w])\\1{5,}"
+            # Common text placeholders
+            - "(?i)(?:s|ex)ample|foo|bar|test|abcd|redacted|key"
+            # Common numeric placeholders
+            - "1234(?:56)"
```

Comment:
I'd say there is no need to check for any potential "56" if we already found 1234 since just 1234 will also match 123456.
```suggestion
            - "1234"
```